modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/suspended/AsyncSyntax.kt

```diff
@@ -0,0 +1,41 @@
+package arrow.effects.typeclasses.suspended
+
+import arrow.Kind
+import arrow.core.Either
+import arrow.core.Left
+import arrow.core.Right
+import arrow.effects.typeclasses.Async
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+import kotlin.coroutines.startCoroutine
+
+interface AsyncSyntax<F> : MonadDeferSyntax<F>, Async<F> {
+
+  override fun <A> f(fa: suspend () -> A): Kind<F, A> =
+    super.async { cb ->
```

Comment:
Why `super`? It's redundant, as MonadDefer doesn't have it.

Reply:
There is another `async` function in this interface that classes with `async` unless you qualify the call like this since it needs to infer the type signature of `cb`.